lib/evm/eei.ts

```diff
@@ -163,10 +161,6 @@ export default class EEI {
    * input data passed with the message call instruction or transaction.
    */
   getCallDataSize(): BN {
-    if (this._env.callData.length === 1 && this._env.callData[0] === 0) {
```

Comment:
Some tests were using `CALLDATASIZE` as a way of determining the address to call. They were failing for the case were calldata was `Buffer<00>` (CALLDATASIZE returned 0 instead of 1).

I'm not sure why this condition was here? any ideas?

Reply:
This is really weird. I checked in pyevm, [and it doesn't have this logic either](https://github.com/ethereum/py-evm/blob/6e9f85b5bd3a206b1a20d32beb6fe812da560903/eth/vm/logic/context.py#L65).